register.js

```diff
@@ -0,0 +1,8 @@
+const tsNode = require('ts-node');
```

Comment:
You don't use this file. So you don't need it. And you have no "ts-node" in your dependencies so it will cause an error.